### ISSUE TITLE
Aria-required property is missing for required field in Navigation screen. 

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#879]
**Describe the bug**
Aria-required property is missing for required field.

**To Reproduce**
1. Go to Navigation screen and select any site to see its navigation.
2. Under Level 2, there is an icon "Add Section". When we click it, a pop up opens. 
3. "Section name" is a required field. Inspect the "Section name" text box element, you will find that the Aria-required=true property is missing for the input text.

**Expected behavior**
Aria-required=true property must be available for all required fields.

**Additional context**
Navigate all the options available for similar issue and fix.
